src/briefcase/commands/create.py

```diff
@@ -368,34 +355,39 @@ def install_app_support_package(self, app: BaseConfig):
             # the support package URL.
             try:
                 support_package_url = app.support_package
-                print("Using custom support package {support_package_url}".format(
-                    support_package_url=support_package_url
-                ))
+                print(
+                    "Using custom support package {support_package_url}".format(
+                        support_package_url=support_package_url
+                    )
+                )
             except AttributeError:
                 support_package_url = self.support_package_url
-                print("Using support package {support_package_url}".format(
-                    support_package_url=support_package_url
-                ))
+                print(
+                    "Using support package {support_package_url}".format(
+                        support_package_url=support_package_url
+                    )
+                )
 
-            if support_package_url.startswith('https://') or support_package_url.startswith('http://'):
+            if support_package_url.startswith(
+                "https://"
+            ) or support_package_url.startswith("http://"):
                 # Download the support file, caching the result
                 # in the user's briefcase support cache directory.
                 support_filename = self.download_url(
                     url=support_package_url,
-                    download_path=Path.home() / '.briefcase' / 'support'
+                    download_path=Path.home() / ".briefcase" / "support",
                 )
             else:
                 support_filename = support_package_url
         except requests_exceptions.ConnectionError:
-            raise NetworkFailure('downloading support package')
+            raise NetworkFailure("downloading support package")
 
         try:
             print("Unpacking support package...")
             support_path = self.support_path(app)
             support_path.mkdir(parents=True, exist_ok=True)
             self.shutil.unpack_archive(
-                str(support_filename),
-                extract_dir=str(support_path)
+                str(support_filename), extract_dir=str(support_path), format="zip"
```

Comment:
If I didn't specify `format="zip"` here, then `unpack_archive` would raise `ValueError` with invalid format.

BTW, what's up with the UUID-style filename that gets created for this file?

Basically... how does this ever work? :)

Reply:
Interesting - omitting the format flag should cause the archive format to be autodetected. That's useful because some of the support packages are tarballs, and some are zip files.

Not sure what you're referring to with the "uuid-style filename", though - the support package should be the name as downloaded from the support folder (i.e. ,"Python-Apple-support.b3.tar.gz"). Are you seeing different behaviour?

Reply:
re: UUID-style filename: It probably has to do with how GitHub chooses URLs for assets. I'll take a look at that.